packages/apps/contracts/CommitRevealApp.sol

```diff
@@ -37,15 +37,15 @@ contract CommitRevealApp {
     bytes32 actionHash;
   }
 
-  function isStateTerminal(AppState state)
+  function isStateTerminal(AppState memory state)
```

Comment:
Is `memory` required now for params?

Reply:
Yes

Reply:
https://solidity.readthedocs.io/en/v0.5.0/050-breaking-changes.html#explicitness-requirements

Reply:
Cool, I like it